migrations/postgres/2018-12-08-175515_constraints/down.sql

```diff
@@ -0,0 +1,25 @@
+-- This file should undo anything in `up.sql`
+ALTER TABLE api_tokens DROP CONSTRAINT IF EXISTS api_tokens_unique_value;
+ALTER TABLE blog_authors DROP CONSTRAINT IF EXISTS blog_author_unique;
+ALTER TABLE blogs DROP CONSTRAINT IF EXISTS blog_unique;
+ALTER TABLE blogs DROP CONSTRAINT IF EXISTS blog_unique_ap_url;
+ALTER TABLE blogs DROP CONSTRAINT IF EXISTS blog_unique_outbox;
+ALTER TABLE blogs DROP CONSTRAINT IF EXISTS blog_unique_inbox;
+ALTER TABLE comments DROP CONSTRAINT IF EXISTS comments_unique_ap_url;
+ALTER TABLE follows DROP CONSTRAINT IF EXISTS follows_unique_ap_url;
+ALTER TABLE instances DROP CONSTRAINT IF EXISTS instance_unique_domain;
+ALTER TABLE likes DROP CONSTRAINT IF EXISTS likes_unique;
+ALTER TABLE likes DROP CONSTRAINT IF EXISTS likes_unique_ap_url;
+ALTER TABLE medias DROP CONSTRAINT IF EXISTS medias_unique_path;
```

Comment:
this constraint won't exist (won't fail because drop if exists, but also useless)